core/.js/src/main/scala/io/udash/bindings/inputs/Select.scala

```diff
@@ -23,7 +23,7 @@ object Select {
     * @param selectModifiers Additional Modifiers for the select tag, don't use modifiers on value, onchange and selected attributes.
     * @return Binding with `select` element, which can be used as Scalatags modifier.
     */
-  def apply[T : PropertyCreator](
+  def apply[T](
```

Comment:
Should `UdashForm` methods: `select`, `multiSelect`, `checkButtons` and `radioButtons` take a `PropertyCreator` instance? Same applies to the `ButtonsComponent` class probably.

Reply:
`PropertyCreator` for `UdashPagination#PageType` seems to be unnecessary - please check.